queries.sql

```diff
@@ -118,3 +118,74 @@ INNER JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.full_name
 ORDER BY animals_owned DESC
 LIMIT 1;
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name
+FROM visits
+INNER JOIN animals ON visits.animal_id = animals.id
+INNER JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT species.name
+FROM visits
+INNER JOIN animals ON visits.animal_id = animals.id
+INNER JOIN species ON species.id = animals.species_id
+INNER JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Stephanie Mendez'
+GROUP BY species.name;
```

Comment:
- [ ] Dear @Bornittah, Good job so far, Here please remember that for this query we need to return the different types of animals that `Stephanie Mendez see` not `species.name`. Please use `animals.name`